_redirects

```diff
@@ -25,6 +25,7 @@ layout: null
 /docs/v21.2/migrate-from-oracle.html /docs/v21.2/migration-overview.html 302
 
 # Renamed pages
+/docs/cockroachcloud/migrations-page.html#upload-a-sql-dump /docs/cockroachcloud/migrations-page.html#upload-file 301
```

Comment:
Redirects don't work for anchors. I'd suggest omitting this line. I can bring this up with Netlify to see if they know of a way to rewrite URLs.

Reply:
Thank you for catching this! I've removed the redirect and added an anchor tag instead.

Reply:
Sure thing! Not that you need my approval, but LGTM tooling-wise!